foundation/foundation-drift-detection/aissemble-foundation-drift-detection-client-python/tests/features/environment.py

```diff
@@ -38,7 +38,7 @@ def before_all(context):
     )
 
     # call to health check to verify Drift Detection Services are started
-    health_check_url = "http://localhost:8084/q/health/live"
+    health_check_url = "http://localhost:8084/invoke-drift/health-check"
```

Comment:
I: The previous healthcheck URL is not a valid endpoint in our deployment. This was causing the ITs to wait the full timeout and eventually print an error before continuing on to successfully execute the actual tests.